packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html

```diff
@@ -45,13 +53,21 @@
                 data-ng-bind=":: formattedDisk"
             ></p>
         </oui-tile-description>
+        <oui-tile-description>
+            <span
+                data-ng-if="$ctrl.upgradeTask.isUpgradeScheduled('storage')"
+                class="oui-badge oui-badge_warning m-0"
+                data-translate="dedicated_server_dashboard_technical_details_upgrade_in_progress"
+                data-translate-values="{ schedule: $ctrl.upgradeTask.scheduledTimeSlot }"
+            ></span>
+        </oui-tile-description>
     </oui-tile-definition>
     <oui-tile-button
-        ng-if="$ctrl.isDisksUpgradable()"
-        on-click="$ctrl.goToManualUpgrade('storage')"
+        ng-if="$ctrl.isDisksUpgradable() || $ctrl.upgradeTask.shouldScheduleUpgrade('storage')"
```

Comment:
move 'storage' to constants. Its been used in multiple places

Reply:
Done